### ISSUE TITLE
In some programs, cannot interrupt reading entire window using portable or temporary copy

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#13343]
### Steps to reproduce:

1. Be running a portable or temporary copy of NVDA.
2. Download the WinAmp 5.8 installer from https://www.winamp.com/ (you don't have to install it to trigger this)
3. Run the installer.
4. Acknowledge the UAC dialog.
5. The first page of the install comes up and starts reading.
6. Press tab / shift / control / other keys.

### Actual behavior:

At step 6, NVDA continues reading the window.  With TAB for instance, the focus DOES move to other elements on the page. 
 If the NVDA highlighter is enabled, the focus highlight follows and also moves to the buttons and links, but NVDA continues to read regardles.

User who reported this to me reports this happens in other places, but this was one example he could easily replicate (I can replicate using NVDA 2021.3.1 and a recent alpha).

### Expected behavior:

Speech should be interrupted as it is in other places. 

#### NVDA version:

Portable NVDA 2021.3.1
Temporary copy of nvda_snapshot_alpha-24639,5b9676a3

#### Windows version:

Windows 10 (64-bit) Version: 21H2 (2009), Build: 19044.1526 

#### Name and version of other software in use when reproducing the issue:

winamp58_3660_beta_full_en-us.exe installer

#### Other information about your system:

This does not repro with an installed copy of NVDA 2021.3.1

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

As noted:

Issue occurs using
Portable NVDA 2021.3.1
Temporary copy of nvda_snapshot_alpha-24639,5b9676a3

Does not repro with an Installed copy of NVDA 2021.3.1

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, a slightly easier way to reproduce this is running Disk Cleanup in admin mode while running portable or temporary NVDA. For several years I thought this was due to portable NVDA not having UI Access privileges (you can sort of get around this by running portable NVDA executable with admin rights). Thanks.

**Comment by cary-rowen** [comment]
Yes, I've been able to reproduce the issue, but I don't know if it's a bug, it would be nice if it could be fixed to make it work as expected.

**Comment by Adriani90** [comment]
cc: @feerrenrut since you've introduced the speech interuption feature in advanced settings.

**Comment by Adriani90** [comment]
I think this needs to be prioritized some how, since people working with portable version get this issue again and again.
cc: @seanbudd

**Comment by seanbudd** [comment]
This likely needs to be closed as invalid.
Portable NVDA cannot read UAC windows as it does not have admin privileges.

Without a valid example of a window which is failed to be read, there is not much we can do here.

**Comment by Adriani90** [comment]
@seanbudd I think the issue here is not that NVDA does not speak the window, but it does not interupt speech when pressing a key on the keyboard even though the interupting speech setting is enabled. When people press tab to navigate through the application they always have to wait until the last focused element is completely reported.

**Comment by Adriani90** [comment]
@Qchristensen , @cary-rowen can you still reproduce this issue also with NVDA 64bit alpha?

**Comment by seanbudd** [comment]
Closing as this is how the UAC dialogs work

**Comment by Qchristensen** [comment]
Note for clarification: The issue here isn't the UAC dialog, but that you are now running the program with elevated privileges, which is a documented limitation of portable copies
